README.md

```diff
@@ -3,17 +3,11 @@
 This project is an extension for VS Code to provide the features of the [Checker Framework](https://checkerframework.org/),
 via the [Checker Framework Language Server](https://github.com/eisopux/checker-framework-languageserver).
 
-
 ## Getting Started
 
-You can either install the extension from the
-[VS Code Extension Marketplace](https://code.visualstudio.com/docs/editor/extension-marketplace)
-or install a locally-built version of the extension.
-Please refer to the "Developer's Guide" below for instructions on how to build from source.
+You can either install the extension from the [VS Code Extension Marketplace](https://code.visualstudio.com/docs/editor/extension-marketplace) or install a locally-built version of the extension.Please refer to the "Developer's Guide" below for instructions on how to build from source.
```

Comment:
Please use line-breaks, as the document did before.
At the moment it is hard to see what exactly changed in this text.
There should definitely be some whitespace before the `Please` sentence.